src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoCategory.java

```diff
@@ -26,8 +26,7 @@
     public MultiblockInfoCategory(IJeiHelpers helpers) {
         this.guiHelper = helpers.getGuiHelper();
         this.background = this.guiHelper.createBlankDrawable(176, 166);
-        ResourceLocation iconLocation = new ResourceLocation(GTValues.MODID, "textures/gui/icon/coke_oven.png");
-        this.icon = this.guiHelper.createDrawable(iconLocation, 0, 0, 16, 16, 16, 16);
+        this.icon = guiHelper.drawableBuilder(GuiTextures.MULTIBLOCK_CATEGORY.imageLocation, 0,0,18,18).setTextureSize(18,18).build();
```

Comment:
It looks like the coding convention is to put space between lists of arguments, so you might want to adjust that here.